### DISCUSSION TITLE
Plan to migrate template to tailwind CSS ?

### DISCUSSION THREAD

**Discussion by cantin-f** [primefaces/community#1174] [PrimeVue Templates] [answered]
I've seen that all blocks will be migrate to Tailwind CSS, do you plan to migrate templates or release new full template using tailwind CSS ?

**Comment by ace-featherlight** [comment]
Me to, will be great if prime-vue have template using tailwind and new unstyle system using tailwind.primeuve preset. 🥳 

I think prime must have new template with latest aura/other preset theme, new design API system, and latest technology & dependency : vite/nuxt, nuxt-tailwind, nuxt-icon, typescript, and all latest feature of vue (and the oldest template will follow up)

**Comment by adiramardiani** [comment]
Hi @cagataycivici @onursenture 

It's nice to see this discussion lively :)

I'm still looking for the right template in the store, but unfortunately of the many navigation models, there is still no mode that has dual menu/two menu/etc, whatever the term is.

Currently I am focusing on ease of navigation for the user, where he can move to pages he has previously opened or in the same context quickly.

https://github.com/primefaces/community/assets/8703419/2f1aa448-5df5-4932-a023-c893f3cae169

This template is free, and functionally it meets my needs, but the ui component used is not primevue, I still can't get away from Primevue :(
- With the mini menu on the side, applications that have lots of menus can be made neater and more organized
- Can also be used to store main modules, such as finance, accounting, warehouse, etc
- The mini menu is scrollable, and can change mode from icon only to icon with text
- The other menu can be pinned, so we can make the menu remain there, or make it minimal
- I think it's a good idea if the Primevue breadcrumb component has a dropdown feature ❤️ 
- The `tab` mechanism for previously opened pages allows users to move very quickly, so so good idea
- I try make MANY tab, and still organized using scroll and next/prev button
- a right-click feature to bring up options from a tab would also be very helpful

I think if Primevue had a menu model like this it on paid template would be amazing, and using tailwindcss to 👍

**Comment by tugcekucukoglu** [comment, marked as answer]
The new version of all templates with Tailwind can be downloaded from the PrimeStore.

**Reply by kamranata** [reply]
Do you plan to upgrade Apollo to tailwind 4?
